mithril-common/src/chain_observer/pallas_observer.rs

```diff
@@ -580,7 +603,7 @@ mod tests {
     }
 
     /// Sets up a mock server.
-    async fn setup_server(socket_path: PathBuf) -> tokio::task::JoinHandle<()> {
+    async fn setup_server(socket_path: PathBuf, intersections: u32) -> tokio::task::JoinHandle<()> {
```

Comment:
Can you add a small comment to explain the way `intersections` work?

Reply:
ofc, done!